packages/ollama-utils/scripts/generate-automap.ts

```diff
@@ -190,7 +201,13 @@ const getSpecialTokens = (tmpl: string): string[] => {
 			.map(() => workerGetTemplate())
 	);
 
+	console.log("====================================");
 	console.log("DONE");
+	console.log("Added templates for:");
+	console.log(addedModels.join("\n"));
+	console.log("Skipped these models due to error:");
+	console.log(skippedModelsDueToErr.join("\n"));
```

Comment:
Can we use console.error() for better error differentiation? This would help separate errors from regular logs, making debugging and log filtering easier.

Reply:
I'm not sure why stderr sometimes come out-of-order on github CI output, so I'm sticking with `console.log` for now